dot/network/transaction.go

```diff
@@ -137,7 +140,16 @@ func (s *Service) startTxnBatchProcessing(txnBatchCh chan *BatchMessage, slotDur
 						continue
 					}
 
-					if !s.gossip.hasSeen(txnMsg.msg) {
+					// TODO: Check if s.gossip.hasSeen should be moved before handleTransactionMessage.
+					// That we could avoid handling the transactions again, which we would have already seen.
```

Comment:
TODO reminder

Reply:
out of scope, created an issue https://github.com/ChainSafe/gossamer/issues/2445

Reply:
Please then mention the issue #2445 in this comment